publish/canisters/BUILD.bazel

```diff
@@ -12,6 +12,7 @@ CANISTERS = {
     "genesis-token-canister.wasm.gz": "//rs/nns/gtc:genesis-token-canister",
     "governance-canister.wasm.gz": "//rs/nns/governance:governance-canister",
     "governance-canister_test.wasm.gz": "//rs/nns/governance:governance-canister-test",
+    "ic-btc-kyt.wasm.gz": "//rs/bitcoin/kyt:btc_kyt_canister",
```

Comment:
@ninegua I took the liberty to add it to the list of published canisters. Is-it ok for you or do you think that's too early?

Reply:
LGTM! Thanks